```
======================================================================
FAIL: test_non_ascii_infile (test.test_json.test_tool.TestTool)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "d:\a\1\s\lib\test\test_json\test_tool.py", line 115, in test_non_ascii_infile
    rc, out, err = assert_python_ok('-m', 'json.tool', infile)
  File "d:\a\1\s\lib\test\support\script_helper.py", line 157, in assert_python_ok
    return _assert_python(True, *args, **env_vars)
  File "d:\a\1\s\lib\test\support\script_helper.py", line 143, in _assert_python
    res.fail(cmd_line)
  File "d:\a\1\s\lib\test\support\script_helper.py", line 70, in fail
    raise AssertionError("Process return code is %d\n"
AssertionError: Process return code is 1
command line: ['d:\\a\\1\\s\\PCbuild\\win32\\python.exe', '-X', 'faulthandler', '-I', '-m', 'json.tool', '@test_4856_tmp']

stdout:
---

---

stderr:
---
'charmap' codec can't decode byte 0x81 in position 10: character maps to <undefined>
---
```

--- a/Lib/test/test_json/test_tool.py
+++ b/Lib/test/test_json/test_tool.py
@@ -107,7 +107,7 @@ class TestTool(unittest.TestCase):
         data = '{"msg": "\u3053\u3093\u306b\u3061\u306f"}'
         expect = textwrap.dedent('''\
         {
-            "msg": "\u3053\u3093\u306b\u3061\u306f"
+            "msg": "\\u3053\\u3093\\u306b\\u3061\\u306f"
         }
         ''').encode()
 
@@ -115,7 +115,7 @@ class TestTool(unittest.TestCase):
         rc, out, err = assert_python_ok('-m', 'json.tool', infile)
 
         self.assertEqual(rc, 0)
-        self.assertEqual(out.splitlines(), expect.encode().splitlines())
+        self.assertEqual(out.splitlines(), expect.splitlines())
         self.assertEqual(err, b'')
 
     def test_infile_outfile(self):

